streams/delete-stream-parameter-data.js

```diff
@@ -0,0 +1,70 @@
+/*
+    This example demonstrates how to delete data for given parameter from specific stream.
+    
+    It uses 2-legged authentication - this requires that application is added to facility as service.
+*/
+import { createToken } from '../common/auth.js';
+import { TandemClient } from '../common/tandemClient.js';
+import { ColumnFamilies, Encoding, QC, getDefaultModel } from '../common/utils.js';
+
+// update values below according to your environment
+const APS_CLIENT_ID = 'YOUR_CLIENT_ID';
+const APS_CLIENT_SECRET = 'YOUR_CLIENT_SECRET';
+const FACILITY_URN = 'YOUR_FACILITY_URN';
+const STREAM_NAME = 'Temperature Sensor';
+const PARAMETER_NAME = 'Temperature';
+
+async function main() {
+    // STEP 1 - obtain token. The sample uses 2-legged token but it would also work with 3-legged token
+    // assuming that user has access to the facility
+    const token = await createToken(APS_CLIENT_ID,
+        APS_CLIENT_SECRET, 'data:read data:write');
+    const client = new TandemClient(() => {
+        return token;
+    });
+    
+    // STEP 2 - get facility and default model. The default model has same id as facility
+    const facilityId = FACILITY_URN;
+    const facility = await client.getFacility(facilityId);
+    const defaultModel = getDefaultModel(facilityId, facility);
+    // STEP 3 - get model schema. It is needed to find parameter id.
+    const schema = await client.getModelSchema(defaultModel.modelId);
+    // STEP 4 - get streams and find stream by name
+    const streams = await client.getStreams(defaultModel.modelId);
+    const stream = streams.find(s => s[QC.Name] === STREAM_NAME);
+
+    if (!stream) {
+        throw new Error(`Stream ${STREAM_NAME} not found`);
+    }
+    // STEP 5 - get stream data
+    const streamKey = Encoding.toFullKey(stream[QC.Key], true);
```

Comment:
There is no need to pass in a flag here. API knows the exact GUID context and can add a flag automatically.

Reply:
I tried this with short key but I got an error (400 - Bad Request) when calling GET timeseries/models/:urn/streams/:streamKey. However it works fine with full key.

Reply:
POST timeseries/models/:modelId/deletestreamsdata also doesn't work with short keys. Did I miss something, pls?

Reply:
I see. It's a bug, Let's hold this PR and fix the API then.